hack/verify-chart.sh

```diff
@@ -0,0 +1 @@
+docker run -it --rm --network host --workdir=/data --volume ~/.kube/config:/root/.kube/config:ro --volume $(pwd):/data quay.io/helmpack/chart-testing:v3.7.0 /bin/bash -c "git config --global --add safe.directory /data; ct install --config=.github/ci/ct.yaml --helm-extra-set-args=\"--set=kind=Deployment --set=podSecurityPolicy.create=false\"" 
```

Comment:
Not necessarily needed in this PR. Though, it would be great to introduce `CONTAINER_ENGINE` var in the [Makefile](https://github.com/openshift/release/blob/master/Makefile#L5) so we can use e.g. `podman` instead of the `docker`.

Reply:
```
$ make ct-helm
./hack/verify-chart.sh
Trying to pull quay.io/helmpack/chart-testing:v3.7.0...
Getting image source signatures
Copying blob 530afca65e2e done  
Copying blob 8fc6dd0270c7 done  
Copying blob 11468c6e8c15 done  
Copying blob 44f8ddf0f4fd done  
Copying blob becf289968d8 done  
Copying blob ef1d4f4b02fd done  
Copying blob 1e4c4ffbf3ee done  
Copying blob dff19b98de5f done  
Copying blob d2ab7f174e47 done  
Copying blob 4f4fb700ef54 done  
Copying config 30a5ebe114 done  
Writing manifest to image destination
Storing signatures
Installing charts...
Error: Error loading configuration: Error loading config file: open .github/ci/ct.yaml: permission denied
Error loading configuration: Error loading config file: open .github/ci/ct.yaml: permission denied
make: *** [Makefile:158: ct-helm] Error 1
```

Reply:
With podman I got this

```
./hack/verify-chart.sh
ERRO[0000] cannot find UID/GID for user lucas: open /etc/subuid: no such file or directory - check rootless mode in man pages. 
WARN[0000] Using rootless single mapping into the namespace. This might break some images. Check /etc/subuid and /etc/subgid for adding sub*ids if not using a network user 
Trying to pull quay.io/helmpack/chart-testing:v3.7.0...
Getting image source signatures
Copying blob 11468c6e8c15 done  
Copying blob ef1d4f4b02fd done  
Copying blob 44f8ddf0f4fd done  
Copying blob 530afca65e2e done  
Copying blob becf289968d8 done  
Copying blob 8fc6dd0270c7 done  
Copying blob 1e4c4ffbf3ee done  
Copying blob d2ab7f174e47 done  
Copying blob dff19b98de5f done  
Copying blob 4f4fb700ef54 done  
Error: writing blob: adding layer with blob "sha256:530afca65e2ea04227630ae746e0c85b2bd1a179379cbf2b6501b49c4cab2ccc": Error processing tar file(exit status 1): potentially insufficient UIDs or GIDs available in user namespace (requested 0:42 for /etc/shadow): Check /etc/subuid and /etc/subgid if configured locally and run podman-system-migrate: lchown /etc/shadow: invalid argument
make: *** [Makefile:158: ct-helm] Error 125
```

Have you tried this with docker? We can introduce that variable and also podman support in a followup PR

Reply:
Ok, it was about running podman with sudo in my case, It just worked after adding sudo (or adding my user to the right group).

Reply:
also, that error does not make much sense :thinking: that file is 644, so it is readable by all :thinking:

Reply:
Selinux was the culprit :)